### ISSUE TITLE
[BUG] Fatal exception: "AttributeError: 'NoneType' object has no attribute 'type'" when running ReplaceReturns transformer

### ISSUE THREAD

**Issue by Ga22be** [MarketSquare/robotframework-tidy#283]
Tool crashes when run with command:
`robotidy --transform ReplaceReturns .`

with config:
```
[tool.robotidy]
overwrite = false
diff = true
check = true
transform = [
    "AddMissingEnd",
    "DiscardEmptySections",
    "InlineIf",
    "MergeAndOrderSections",
    "NormalizeNewLines",
    "NormalizeSectionHeaderName",
    "NormalizeSettingName",
    "OrderSettingsSection",
    "RemoveEmptySettings",
    "RenameTestCases",
    "ReplaceBreakContinue",
    # "OrderSettings",
]
configure = [
    # "OrderSettings:keyword_before=tags,documentation,arguments,timeout",
    # "OrderSettings:test_before=template,tags,documentation,timeout,setup"
]
```

**Stack trace**
```
me@host:~/root (robotidy *%)$ robotidy --transform ReplaceReturns .
Loaded configuration from /my/home/root/robotidy.toml
Traceback (most recent call last):
  File "/my/home/.local/bin/robotidy", line 8, in <module>
    sys.exit(cli())
  File "/usr/lib/python3/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/me/.local/lib/python3.8/site-packages/robotidy/decorators.py", line 63, in wrapper
    raise err
  File "/home/me/.local/lib/python3.8/site-packages/robotidy/decorators.py", line 46, in wrapper
    return func(*args, **kwargs)
  File "/home/me/.local/lib/python3.8/site-packages/robotidy/cli.py", line 391, in cli
    status = tidy.transform_files()
  File "/home/me/.local/lib/python3.8/site-packages/robotidy/app.py", line 64, in transform_files
    diff, old_model, new_model = self.transform(model)
  File "/home/me/.local/lib/python3.8/site-packages/robotidy/app.py", line 84, in transform
    transformer.visit(model)
  File "/home/me/.local/lib/python3.8/site-packages/robotframework-4.0-py3.8.egg/robot/parsing/model/visitor.py", line 62, in visit
    return visitor(node)
  File "/usr/lib/python3.8/ast.py", line 447, in generic_visit
    value = self.visit(value)
  File "/home/me/.local/lib/python3.8/site-packages/robotframework-4.0-py3.8.egg/robot/parsing/model/visitor.py", line 62, in visit
    return visitor(node)
  File "/usr/lib/python3.8/ast.py", line 447, in generic_visit
    value = self.visit(value)
  File "/home/me/.local/lib/python3.8/site-packages/robotframework-4.0-py3.8.egg/robot/parsing/model/visitor.py", line 62, in visit
    return visitor(node)
  File "/home/me/.local/lib/python3.8/site-packages/robotidy/transformers/ReplaceReturns.py", line 66, in visit_Keyword
    return_stmt = create_statement_from_tokens(
  File "/home/me/.local/lib/python3.8/site-packages/robotidy/utils.py", line 308, in create_statement_from_tokens
    return statement([indent, Token(statement.type), *tokens])
AttributeError: 'NoneType' object has no attribute 'type'
Fatal exception occurred. You can create an issue at https://github.com/MarketSquare/robotframework-tidy/issues . Thanks!

```

**Comment by bhirsz** [comment]
Hi, thanks for raising the issue. It looks similiar to  #283 - what's your Robot Framework version?

**Comment by bhirsz** [comment]
I apologise, I meant #281. ``ReplaceReturns`` is only supported from RF 5.0 onward. If you want to use it you need to upgrade your installed Robot Framework. But of course it's still a bug since like I mentioned in the linked issue - Robotidy should warn user when trying to use transformers not compatible with their installed Robot Framework version.

**Comment by Ga22be** [comment]
@bhirsz, that seems like a bug yes. I have similar issues with _InlineIf_ now on a different set of robot-files. We are not able to update to RF5.0 currently, so it would be nice if all options for robotidy became RF-version aware :)

**Comment by bhirsz** [comment]
Sure! They're actually aware but there is a bug where ``--transform`` or ``enabled`` overwrite it. I will fix it and let you know.

There is also a separate issue #253 where the goal is to be able to use Robotidy to transform with specific RF version set (ie you can run robotidy on RF5.0 but transform with 4.0 as a goal).

**Comment by Ga22be** [comment]
Sounds great, I'll be waiting ;)

**Comment by bhirsz** [comment]
Released in Robotidy 2.2.
